### ISSUE TITLE
Does bbin support installing multiple libraries from `bb.edn`?

### ISSUE THREAD

**Issue by teodorlu** [babashka/bbin#59]
Since `:bbin/bin` is a map, I assumed installing multiple binaries was okay:

```clojure
{:deps {io.github.teodorlu/shed {:local/root "."}}
 :bbin/bin {update-repos {:main-opts ["-m" "teodorlu.shed.update-repos/-main"]}
            browsetxt {:main-opts ["-m" "teodorlu.shed.browsetxt/-main"]}}}
```

([source](https://github.com/teodorlu/shed/tree/47c1d12468177cfe8ffa2957b79047d35bb45bb5/bb.edn))

But this seems to only install `update-repos`.

----

Should bbin be able to install multiple binaries, and not just one?

**Comment by borkdude** [comment]
I think this could be improved. PR welcome if @rads agrees.

**Comment by teodorlu** [comment]
The first key-value pair is picked in a `(some-> bin-config first key str)` here. Looks like the rest of the `install-deps-git-or-local` function assumes that `script-name` is _one script_ and not multiple scripts.

https://github.com/teodorlu/bbin/blob/6b778266a490a4d0d89618a60bffc9617f62e7e3/src/babashka/bbin/scripts.clj#L413-L417

**Comment by rads** [comment]
Yep, the map is intentional to support multiple scripts, though I haven't gotten around to implementing that quite yet. See also:
- https://github.com/babashka/bbin/issues/18
